bench_async_fetch_test.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/dgraph-io/badger"
+	"github.com/dgraph-io/badger/y"
+)
+
+func BenchmarkIterateValueFetch(b *testing.B) {
+	bdb, err := getBadger()
+	y.Check(err)
+	defer bdb.Close()
+	k := make([]byte, 1024)
+	v := make([]byte, Mi)
+	b.ResetTimer()
+
+	b.Run("iterate-async-fetch", func(b *testing.B) {
+		for j := 0; j < b.N; j++ {
+			var count int
+			opt := badger.IteratorOptions{}
+			opt.PrefetchSize = 10000
```

Comment:
Set this to 256. Also, ensure that gomaxprocs is 128.

Reply:
Why 256?